pages/workspace/tracking-plan/publishing.mdx

```diff
@@ -448,13 +448,14 @@ Json representation of your tracking plan, following the [Json Schema](https://j
 
 ###### PropertyRule Model
 
-| Attribute    | Type               | Description                                   |
-| :----------- | :----------------- | :-------------------------------------------- |
-| description  | string             | Property description                          |
-| type         | string             | Property type                                 |
-| nameMappings | array(NameMapping) | Name mappings defined for given property      |
-| ?enum        | array(string)      | Allowed enum values                           |
-| ?operations  | array(string)      | Array of property ops, e.g. "SnowplowContext" |
+| Attribute    | Type                         | Description                                   |
+| :----------- | :--------------------------- | :-------------------------------------------- |
+| description  | string                       | Property description                          |
+| type         | string                       | Property type                                 |
+| nameMappings | array(NameMapping)           | Name mappings defined for given property      |
+| ?const       | Depends on the property type | Pinned property value                         |
```

Comment:
This line is the only change, rest is formatting fix

Reply:
Is it auto format? What tool are you using?

Reply:
Hmmm not sure if I'm using anything specific 🤔